MAINTAINERS.md

```diff
@@ -0,0 +1,55 @@
+## Maintainers
+Documentation for maintainers.
+
+Current maintainers: 
+- [Jake Bolam](https://github.com/jakebolam)
+- [Maximilian Berkmann](https://github.com/Berkmann18) 
+- [Tyler Benning](https://github.com/tbenning) 
+- [Jeff Wen](https://github.com/sinchang)
+
+See [the github All Contributors Core Team group](https://github.com/orgs/all-contributors/teams/core/members)
+
+## Roadmap/Goals
+### Unite all contributor efforts ✅
+- [x] Create central org, and move across all-contributors, all-contributors-cli, all-contributors-atom and all-contributors-bot
+- [x] Add Branding, central Communication Places
+- [x] Launch website and centralize documentation
+
+### Establish an All Contributors community
+- [ ] Establish a check-in ritual
+- [ ] Establish a culture of gratitude ritual 
+- [ ] Develop more proactive information-sharing practices
+- [ ] Develop more open and inclusive decision-making processes
+
+### Building an All Contributors presence and visibilty
+- [x] Switch to .org from .js.org and promote to communities outside JavaScript
+- [ ] Create a video
+- [ ] Improve SEO (see goals)
+- [ ] Add all contributors bot to github marketplace (see goals 250 app installs)
+
+
+## Metrics OKR Period - Open Leaders (Mid Feb 2019 until April 2019)
+### Project & Website
+Date | Maintainers | Contributors |Stars | Website Languages  | Website Sessions | Seo Hits
+---|---|---|---|---|---|---
+**TARGET for April 201** | 4 | 40 | 2700 | 10 | 500 | 10
+Wed Feb 20th | 4 | 34 | 2442 | 7 | 382 | 0
+Wed Feb 13th | 4 | 33 | 2430 | 6 | 431 | 0
+
+
+### Bot
+Date | Bot Installs | Bot Stars | Bot Messages | Bot PRs 
+---|---|---|---|---
+**TARGET for April 2019** | 250 | 100 | 50 | 40
+Wed Feb 20th | 107 | 75 | 21 | 20
+Wed Feb 13th | 82 | 71 | 24 | 23
```

Comment:
From @gr2m.

Should we track uninstalls of the bot?

Reply:
It would be useful to know.

Reply:
Ticketed: https://github.com/all-contributors/all-contributors-bot/issues/148